### PR TITLE
Fix broken polymer-2-element serve page

### LINKED ISSUE #610: Problems with polymer-2-element template
_From @nicolasr75 on March 14, 2017 18:25_

### Description
The polymer-2-element template does not work because of wrong paths.

#### Steps to Reproduce

Do _polymer init_

Select the _polymer-2-element_ template

Do _polymer serve_ as described in the readme

Start a browser and go to http://localhost:8081

#### Expected Results

The browser should load the page and display _Hello my-elem_
(because I selected my-elem as the element name in the wizard).

#### Actual Results

The page stays blank and the dev tools console shows

GET http://localhost:8081/webcomponentsjs/webcomponents-loader.js 404 (Not Found)
GET http://localhost:8081/iron-component-page/iron-component-page.html 404 (Not Found)

Looking at index.html one can see that the generated paths are missing the bower_components directory. F.e. the generated path to webcomponents is:
../webcomponentsjs/webcomponents-loader.js
 (Note: in the application template this works because the paths are correct there!)

### Browsers Affected

- [x] Chrome
- [x] Firefox
- [x] Edge
- [x] Safari 9
- [x] Safari 8
- [x] IE 11

### Versions

- Polymer: v0.18.0-pre.13
- webcomponents: v1.0.0.-rc.5

_Copied from original issue: Polymer/polymer#4430_

### PR DESCRIPTION
**Pull request by FredKSchott**
- [X] CHANGELOG.md has been updated
- Fixes #610

### CODE REVIEW

**Comment by aomarks** [comment]
Review status: 0 of 3 files reviewed at latest revision, 1 unresolved discussion.

---

*[CHANGELOG.md, line 6 at r1](https://reviewable.io:443/reviews/polymer/polymer-cli/652#-KhPGam9P-i-T7cyDhYY:-KhPGam9P-i-T7cyDhYZ:b-eejwo6) ([raw file](https://github.com/polymer/polymer-cli/blob/8da622166540e16b466c4d6052acfa762d8c6209/CHANGELOG.md#L6)):*
> ```Markdown
> 
> - `build`: Fixed a bug where `--insert-prefetch-links` would generate 404ing imports.
> - `init`: For `polymer-2-element` template, remove iron-component-page from the served `index.html` file.
> ```

Maybe a little more context? Why was it removed (because it's broken/deprecated I assume?)

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/polymer/polymer-cli/652)*

File: src/init/element/templates/polymer-2.x/index.html [hunk 1]
```diff
@@ -2,15 +2,15 @@
...
+    <meta http-equiv="refresh" content="0;url=demo/" />
     <title><%= name %></title>
-
-    <script src="../webcomponentsjs/webcomponents-loader.js"></script>
-
-    <link rel="import" href="../iron-component-page/iron-component-page.html">
   </head>
   <body>
-    <iron-component-page src="<%= name %>.html"></iron-component-page>
+    <!--
```
Comment (justinfagnani) [inline src/init/element/templates/polymer-2.x/index.html]: This will result in a blank page, maybe make the notice not a comment

Comment (FredKSchott) [inline src/init/element/templates/polymer-2.x/index.html]: See `<meta>` tag above, when served / viewed in a browser will immediately redirect to the demo. This comment is for anyone viewing in their editor.

**Comment by FredKSchott** [comment]
@aomarks good idea, added a reason.

@aomarks @justinfagnani PTAL

**Pull request merged**